docs/supported/index.rst

```diff
@@ -188,17 +188,16 @@ Supported Operating Systems
 	        <a href="../index.html" class="button">Learn more</a>
 	    </div>
 	    <div class="pod-boxout pod-boxout--4up pod-boxout--orchestration">
+			<img src="../_static/images/rhel72x.png" alt="RHEL 7 logo"/>
+			<span>RHEL 7</span>
+	        <a href="../index.html" class="button">Learn more</a>
```

Comment:
❓ Not specific to this change, but all the supported configurations (except CoreOS) link to back to the beginning of the docs - is this intended?